sass/_Variables.scss

```diff
@@ -71,19 +71,19 @@ $default-border: 1px solid $color-light-grey;
 }
 
 @mixin iconWithHoverState($iconBase, $float:'left', $inactive : 'coveo-disabled') {
-  $iconActive: #{$iconBase} + '-active';
+  $iconActive: '#{$iconBase}-active';
   .coveo-icon {
-    @extend #{$iconBase};
+    @extend #{$iconBase} !optional;
     float: #{$float};
   }
   &:hover {
     .coveo-icon {
-      @extend #{$iconActive};
+      @extend #{$iconActive} !optional;
```

Comment:
Wow, there was a typo when including the mixin somewhere: `dropwdown` instead of `dropdown`. 🙈